libibverbs/cmd_qp.c

```diff
@@ -383,7 +383,7 @@ int ibv_cmd_create_qp(struct ibv_pd *pd,
 	int ret;
 
 	memcpy(&attr_ex, attr, sizeof(*attr));
-	attr_ex.comp_mask |= IBV_QP_INIT_ATTR_PD;
```

Comment:
Hmm, it is because of padding, please instead open code each assignment instead of using the memcpy.

Reply:
Ah yes, I missed that. Updated.